__tests__/app.test.js

```diff
@@ -284,4 +284,48 @@ describe('POST/api/articles/:article_id/comments', () => {
         expect(response.body.msg).toBe("bad request")
       })
     });
+    test('STATUS - 400: should return appropriate error when passed invalid key in body', () => {
+      return request(app)
+      .post("/api/articles/2/comments")
+      .send({username: "rogersop", comment: "Very interesting"})
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("bad request")
+      })
+    });
+    test('STATUS - 400: should return appropriate error when passed a body with too many keys', () => {
```

Comment:
This is only passing because of a count on comment keys, this isn't checking which keys should be used.